examples/triangle.py

```diff
@@ -89,13 +89,13 @@ def setup_draw(context, device):
         vertex={
             "module": shader,
             "entry_point": "vs_main",
-            "buffers": [],
-        },
-        primitive={
-            "topology": wgpu.PrimitiveTopology.triangle_list,
-            "front_face": wgpu.FrontFace.ccw,
-            "cull_mode": wgpu.CullMode.none,
         },
+        # primitive elided, since we are using its default value and want to test that.
```

Comment:
I understand what you're trying to say and do here, but I don't think this should be merged into the examples. Just remove the commented code.